server/gqlcorp/typeMain.js

```diff
@@ -135,7 +135,8 @@ const typeCreator = (nonJoinTables, fktObj, fktObjNoJoins, nullable) => {
           // but if it's in the foreign key object, it adds the type as a connection to a different table.
           const temp = fktObj[key][prop]
           // const temp = camelCase(fktObj[key][prop]);
-          typeObj[key][temp] = '['+capFirstLet(singular(temp))+']';
+          // typeObj[key][temp] = '['+capFirstLet(singular(temp))+']';
+          typeObj[key][temp] = '['+temp.slice(0,1).toUpperCase()+camelCase(singular(temp)).slice(1)+']';
```

Comment:
What is the purpose of slice here?

Reply:
should be pascal function use, resolve later